src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/App.razor

```diff
@@ -5,7 +5,7 @@
     </Found>
     <NotFound>
         <LayoutView Layout="@typeof(MainLayout)">
-            <p>Sorry, there's nothing at this address.</p>
+            <p role="alert">Sorry, there's nothing at this address.</p>
```

Comment:
Adding `role="alert"` to these particular messages is important because otherwise, after you navigate to one of them, nothing at all would happen and a screen-reader user may not know what's going on. With `role="alert"`, the screen reader will announce this.